introspection/README.md

```diff
@@ -5,20 +5,12 @@
 - State: 
   - Spec: In Progress 🚧
   - Implementation: Unknown ❔
-
-When a user gets started with Prisma in a brownfield project, they can let Prisma introspect their database to generate the initial Prisma data model definition.
```

Comment:
We actually "require" this introduction summary sentence - so we should come up with something more generalized about "Introspection" in the Prisma context. (Matt might have something liek that already in his Prisma framework PR)